service/client.go

```diff
@@ -72,4 +72,14 @@ type Client interface {
 
 	// Returns whether we attached to a running process or not
 	AttachedToExistingProcess() bool
+
+	// Returns concrete location information described by a location expression
```

Comment:
Need to update documentation for `break <line_number>` support that I saw you added.